libcloudforensics/gcp.py

```diff
@@ -723,14 +724,15 @@ def CreateImageFromDisk(self, src_disk, name=None):
             name,
         'sourceDisk':
             'projects/{project_id}/zones/{zone}/disks/{src_disk}'.format(
-                project_id=self.project_id, zone=src_disk.zone,
+                project_id=src_disk.project.project_id, zone=src_disk.zone,
                 src_disk=src_disk.name)
     }
     gce_image_client = self.GceApi().images()
-    request = gce_image_client.insert(project=self.project_id, body=image_body)
+    request = gce_image_client.insert(
+        project=self.project_id, body=image_body, forceCreate=True)
     response = request.execute()
     self.BlockOperation(response)
-    return GoogleComputeImage(self.project_id, None, name)
+    return GoogleComputeImage(self, None, name)
```

Comment:
Why is zone `None` here?

Reply:
yes, Images are global resources in GCP and zone should be None, in addition any mutating operation will directly create an GlobalOperation object.